doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Sonata project contribution
+
+Thanks for you interest onto Sonata projects!
+
+## Summary
+
+* [Issues]()
+* [Pull Requests](#pull-requests)
+* [Label rules]()
+
+## Pull Requests
+
+All the sonata team will be glad to review your code changes propositions! :smile:
+
+But please, read the following before.
+
+### Coding style
+
+Each project follows [PSR-1](http://www.php-fig.org/psr/psr-1/), [PSR-2](http://www.php-fig.org/psr/psr-2/)
+and [Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html) for coding style,
+[PSR-4](http://www.php-fig.org/psr/psr-4/) for autoloading.
+
+Please [install PHP Coding Standard Fixer](http://cs.sensiolabs.org/#installation)
+and run this command before committing your modifications:
+
+```bash
+php-cs-fixer fix --verbose
+```
+
+### Writing a Pull Requests
```

Comment:
I recently added a PR and it would be nice to have an instruction about how to "rework/update PRs" (see [Submitting a Patch (Symfony)](https://symfony.com/doc/current/contributing/code/patches.html#rework-your-patch)) .
Especially helpful would be the part: Please squash your commits (and look here on how to do that).